### PR TITLE
added script usage to package-installation page

### PR DESCRIPTION
**Pull request by eLtMosen**
Suggestion to include a short script usage description first to not give newcomers the impression the manual method is needed.

### CODE REVIEW

File: pages/wiki/package-installation.md [hunk 1]
```diff
@@ -7,13 +7,25 @@ layout: documentation
...
+    <p>To use the script you will need to clone the unofficial-watchfaces repo.
+      <pre><code>git clone https://github.com/AsteroidOS/unofficial-watchfaces</code></pre>
+      Change into the new directory.
+      <pre><code>cd unofficial-watchfaces</code></pre>
+      Execute the script with no flag to use SSH connection and SCP commands.
+      <pre><code>./deploy.sh</code></pre>
+      Or use ADB connection and commands with the -a flag.
+      <pre><code>./deploy.sh -a</code></pre></p>
+  </div>
+  <h4>Manual method</h4>
```
Comment (jrtberlin) [inline pages/wiki/package-installation.md:23]: I would add a line break above the `<h4>` element for the looks.

Currently:
without_linebreak

With `<br/>`:
with_linebreak

**Review by jrtberlin — CHANGES_REQUESTED** [review]
I think this is a good improvement overall. With an additional line break I say lgtm!

File: pages/wiki/package-installation.md (hunk 1 above)
Comment (eLtMosen) [reply pages/wiki/package-installation.md:23]: Absolutely. 
Tbh i did this in blind flight and just copied the code from the block below.
How can i have an environment to see what i even do in Less? Any hint what to setup/install?

Comment (jrtberlin) [reply pages/wiki/package-installation.md:23]: No worries. Catching things like this is the purpose of reviews!
I just did `hub pr checkout 113 && npx grunt && chromium _asteroidos.org/wiki/package-installation/index.html` to review the pr.

**Comment by jrtberlin** [comment]
Thank you!

**Pull request merged**